src/chainstate/burn/db/sortdb.rs

```diff
@@ -3052,14 +3064,15 @@ impl<'a> SortitionHandleTx<'a> {
             &block_commit.key_block_ptr,
             &block_commit.key_vtxindex,
             &to_hex(&block_commit.memo[..]),
-            &burn_fee_str,
+            &block_commit.burn_fee.to_string(),
             &tx_input_str,
             sort_id,
             &serde_json::to_value(&block_commit.commit_outs).unwrap(),
+            &block_commit.sunset_burn.to_string(),
         ];
 
-        self.execute("INSERT INTO block_commits (txid, vtxindex, block_height, burn_header_hash, block_header_hash, new_seed, parent_block_ptr, parent_vtxindex, key_block_ptr, key_vtxindex, memo, burn_fee, input, sortition_id, commit_outs) \
-                      VALUES (?1, ?2, ?3, ?4, ?5, ?6, ?7, ?8, ?9, ?10, ?11, ?12, ?13, ?14, ?15)", args)?;
+        self.execute("INSERT INTO block_commits (txid, vtxindex, block_height, burn_header_hash, block_header_hash, new_seed, parent_block_ptr, parent_vtxindex, key_block_ptr, key_vtxindex, memo, burn_fee, input, sortition_id, commit_outs, sunset_burn) \
```

Comment:
Not saying it's wrong, but is there a particular reason why `sunset_burn` is its own column, instead of just being added to the `burn_fee`?

Reply:
Yes -- it's because we apply the sunset ratio checks to a LeaderBlockCommitOp after construction (through the check method), so that struct needs to separate the burn_fee from the sunset_burn. While this check has already been performed by the point that it is being inserted here, these tables serialize the whole struct, so it seemed appropriate to keep that field in the table.